move/axelar_gateway/sources/gateway.move

```diff
@@ -323,6 +355,29 @@ public fun take_approved_message(
     )
 }
 
+// --------------
+// Ticket Getters
+// --------------
```

Comment:
do we need these to be public or test only is fine? can the gas service just emit the ticket object (instead of retrieving each field)?

Reply:
Only if the ticket objet has `copy` and `drop` which I think it should not have. Thought of maybe adding a separate module for the type, but that was a bit ugly as well, because then the gateway itself could not access the fields easily (it could call destroy and capture all of them but having the fields be named is easier). Not sure which I prefer.